pkg/config/config_template.yaml

```diff
@@ -1938,20 +1938,6 @@ api_key:
 
 {{ end -}}
 
-{{- if .DataStreamsModule }}
```

Comment:
Seems like it was added 8 months ago (https://github.com/DataDog/datadog-agent/pull/15430) so it has been part of multiple releases, do you really want to remove it ?

Reply:
yes, it was approved by @guyarb as well

Reply:
the feature was part of a partnership with DSM, and it was categorized as private beta.
We paused the partnership and currently we have no customer in the private beta.